openapi.yml

```diff
@@ -658,6 +667,36 @@ components:
           "registration_status": "PRE_DRAFT",
```

Comment:
is this registration_status here still used ?

Reply:
It's still used in other parts of the code, which still need to be changed. This PR is a partial work because more changes are needed to other methods of organization controller and service, but I preferred to create the PR already otherwise it would become too big to be reviewed.